src/modals/import-account-modal/import-account-seed-form/import-account-seed-form.tsx

```diff
@@ -0,0 +1,96 @@
+import { Formik } from 'formik';
+import React, { Dispatch, FC, SetStateAction } from 'react';
+import { View } from 'react-native';
+
+import { ButtonLargePrimary } from '../../../components/button/button-large/button-large-primary/button-large-primary';
+import { ButtonLargeSecondary } from '../../../components/button/button-large/button-large-secondary/button-large-secondary';
+import { ButtonsContainer } from '../../../components/button/buttons-container/buttons-container';
+import { Divider } from '../../../components/divider/divider';
+import { InsetSubstitute } from '../../../components/inset-substitute/inset-substitute';
+import { Label } from '../../../components/label/label';
+import { ScreenContainer } from '../../../components/screen-container/screen-container';
+import { RadioButton } from '../../../components/styled-radio-buttons-group/styled-radio-buttons-group';
+import { FormMnemonicInput } from '../../../form/form-mnemonic-input';
+import { FormPasswordInput } from '../../../form/form-password-input';
+import { FormRadioButtonsGroup } from '../../../form/form-radio-buttons-group';
+import { FormTextInput } from '../../../form/form-text-input';
+import { ImportAccountDerivationEnum, ImportAccountSeedValues } from '../../../interfaces/import-account-type';
+import { useShelter } from '../../../shelter/use-shelter.hook';
+import { useAccountsListSelector } from '../../../store/wallet/wallet-selectors';
+import { formatSize } from '../../../styles/format-size';
+import {
+  importAccountSeedFormInitialValues,
+  importAccountSeedFormValidationSchema
+} from '../import-account-modal.form';
+import { useImportAccountModalStyles } from '../import-account-modal.styles';
+
+interface Props {
+  importAccountStep: number;
+  setImportAccountStep: Dispatch<SetStateAction<number>>;
+}
+
+export const ImportAccountSeedForm: FC<Props> = ({ setImportAccountStep, importAccountStep }) => {
+  const styles = useImportAccountModalStyles();
+  const { createImportedAccountWithSeed } = useShelter();
+  const accountsLength = useAccountsListSelector().length + 1;
+  const derivationRadioButtons: RadioButton<ImportAccountDerivationEnum>[] = [
+    { value: ImportAccountDerivationEnum.DEFAULT, label: 'Default account (the first one)' },
+    { value: ImportAccountDerivationEnum.CUSTOM_PATH, label: 'Custom derivation path' }
+  ];
+
+  const onSubmit = (values: ImportAccountSeedValues) =>
+    createImportedAccountWithSeed({
+      name: `Account ${accountsLength}`,
+      seedPhrase: values.seedPhrase,
+      password: values.password,
+      derivationPath: values.derivationPath
```

Comment:
check here for `derivationType` value as well. (User could change `derivationPath` and switch back to `ImportAccountDerivationEnum.DEFAULT`)

Reply:
and there will be no value in `derivationPath`, will be used default value